presentation/src/main/java/com/yapp/presentation/ui/AttendanceScreen.kt

```diff
@@ -91,7 +93,17 @@ fun AttendanceScreen(
 
         composable(
             route = AttendanceScreenRoute.HELP.route
-        ) {}
+        ) {
+        }
+
+        composable(
+            route = AttendanceScreenRoute.NAME.route
+        ) {
+            Name(
+                onClickBackBtn = { navController.navigate(AttendanceScreenRoute.LOGIN.route) },
```

Comment:
```suggestion
                onClickBackBtn = { navController.popBackStack() },
```
어짜피 이름 작성 화면 아래에 로그인 화면이 깔려있을테니 Pop으로 빼내는게 더 좋을 것 같아요!